content/pages/2023/current_sponsors.md

```diff
@@ -9,7 +9,27 @@ slug: sponsors_2023
 
 # Current Sponsors
 
-TBA
+## Gold Sponsors
+
+<table style="table-layout:fixed">
+    <tr>
+        <td style="vertical-align: middle; width: 30%; padding-right:10px; padding-left:10px;">
+            <img src="../static/2023/sponsors/sponsor_logos/Roche_Logo.svg">
+        </td>
+        <td style="text-align: left">
+            Throughout its 125-year history, <a href="https://www.roche.com/">Roche</a>
+            has grown into one of the world’s largest biotech companies,
+            as well as a leading provider of in-vitro diagnostics and a global
+            supplier of transformative innovative solutions across major disease areas.
+            At <a href="https://www.roche.ch/en/ueberuns/pharma-research-and-early-development.htm">
+Pharma Research & Early Development (pRED) in Basel</a>, it has
+            more than 1800 dedicated scientists, clinicians, engineers and
+            technicians from almost 60 nations, sharing the passion for
+            developing and bringing new and effective medicines to patients.
+          </td>
+    </tr>
+</table>
+
 
 ## Become a Sponsor
-See guidelines on how to become a sponsor [here](sponsoring.md).
+See guidelines on how to become a sponsor [here](sponsoring.html).
```

Comment:
@SepidehAlassi, I think `sponsoring.html` doesn't exist, however the `sponsoring.md` does: https://github.com/euroscipy/euroscipy.github.io/blob/feature/sponsoring/content/pages/2023/sponsoring.md